esp_components/input-hal/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+FILE(GLOB SRCS "${CMAKE_CURRENT_SOURCE_DIR}/../../device/esp32/input/*.cpp")
```

Comment:
are you able to do `${PROJECT_SOURCE_DIR}` insstead of `CURRENT_SOURCE_DIR/../..`